src/utils.ts

```diff
@@ -35,3 +42,7 @@ export function generateSvgUri(
   const dataUri = `data:image/svg+xml;base64,${btoa(svgContent)}`;
   return vscode.Uri.parse(dataUri);
 }
+
+export function setStatusBarMessage(message: string) {
+  vscode.window.setStatusBarMessage(`LOG ANALYSIS: ${message}`, 5000);
+}
```

Comment:
Do you want to use this more? Now it is just used 1 time.

Reply:
I know but this will be useful in the future.

Reply:
got it